packages/atomic/src/components/result-lists/atomic-result-children/atomic-result-children.tsx

```diff
@@ -230,7 +237,16 @@ export class AtomicResultChildren {
           part="show-hide-button"
           class="show-hide-button"
           style="text-primary"
-          onClick={() => (this.hideResults = !this.hideResults)}
+          onClick={() => {
+            if (this.hideResults) {
+              this.foldedResultList.logShowMoreFoldedResults(
```

Comment:
If the event is called when loading a collection, I don't understand why we re-send the event here ?

Doesn't it get sent two times to the API then ?